csharp/ql/test/query-tests/Security Features/CWE-079/XSSRazorPages/gen_files.py

```diff
@@ -0,0 +1,46 @@
+# A script for generating code from .cshtml files, mimicking the output of the C# compiler with an option that is not available from the codeql test runner.
```

Comment:
How do I invoke this script properly? :-)

Reply:
What exceptions are you getting?

Reply:
It appears that one need to provide the full path to script when executing it as it is needed for the working dir.
The following works for me:
```
python3 /Users/michaelnebel/Work/semmle-code/ql/csharp/ql/test/query-tests/Security\ Features/CWE-079/XSSRazorPages/gen_files.py
``
Where as (when standing in the directory with the script) doesn't work.
```
python3 gen_files.py
```
Maybe it would be worth adding a couple of print statements to the script to get an indication where files are written and also how to invoke the script (maybe `/ql/src/Stubs/make_stubs_nuget.py` could provide some inspiration)